.github/workflows/housekeeping-lock-threads.yml

```
@@ -0,0 +1,44 @@
+name: "Housekeeping / Lock Threads"
+
+on:
+  schedule:
+    - cron: '*/15 * * * *'
+
+concurrency:
+  group: lock-threads
```

Same will be used for stale 
```suggestion
  group: housekeeping
```